packages/pretty-proptypes/src/PrettyConvert/converters.test.js

```diff
@@ -124,35 +124,37 @@ test('resolve generic of array', () => {
 
 test('objects with null members do not throw', () => {
   const type = {
-    kind: "object",
+    kind: 'object',
     members: [
       {
-        kind: "property",
+        kind: 'property',
         optional: false,
         key: {
-          kind: "id",
-          name: "children"
+          kind: 'id',
+          name: 'children'
         },
         value: {
-          kind: "generic",
+          kind: 'generic',
           value: {
-            kind: "id",
-            name: "React.ReactNode"
+            kind: 'id',
+            name: 'React.ReactNode'
           }
         }
       },
       null
     ],
-    referenceIdName: "LabelTextProps"
-  }
+    referenceIdName: 'LabelTextProps'
+  };
   expect(() => prettyConvert(type, components)).not.toThrow();
-})
+});
 
+// eslint-disable-next-line jest/no-disabled-tests
 test.skip('object with spread object', () => {
-  let values = getSingleDefault(`{ ...something, c: 'something' }`);
```

Comment:
Does it matter that we're losing this? (I don't think it does but just wanted to flag it)

Reply:
Not that i know of, seems to be unused by that test. Good chance it's some copy-pasta haha!